src/controller/trainee/controller.ts

```diff
@@ -0,0 +1,68 @@
+import { Router, Request, Response, nextFunction } from "express";
+import { successHandler } from "./";
+class TraineeController {
+  private static instance: TraineeController;
+  public static getInstance() {
+    if (!TraineeController.instance) {
+      TraineeController.instance = new TraineeController();
+    }
+    return TraineeController.instance;
+  }
+  get(req: Request, res: Response, next: nextFunction) {
```

Comment:
There is no use of next function here inside the controller.
Also, if it is required use the proper type i.e NextFunction